docs/cloud/details/access_management/access_management.md

```diff
@@ -15,7 +15,7 @@ When a data source is synchronized, the existing access controls are imported in
 
 [![Access Controls](/assets/images/Access_Controls.jpg)](/assets/images/Access_Controls.jpg){:target="_blank"}
 
-On one side, the Access Control points to the Data Objects it applies to (the `WHAT`) and on the other side it points to `WHO` it applies to. The exact meaning of these links, and which links are possible depends on the Access Control action.
+On one side, the Access Control points to the Data Objects it applies to (the `WHAT`) and on the other side it points to `WHO` it applies to. The exact meaning of these links, and which links are possible depends on the Access Control action (and Grant Category).
```

Comment:
Just wondering is do we explain the term "Grant Category" somewhere?

Reply:
We do in the concepts